pkg/virtctl/imageupload/imageupload.go

```diff
@@ -346,9 +346,13 @@ func waitUploadPodRunning(client kubernetes.Interface, namespace, name string, i
 }
 
 func createUploadPVC(client kubernetes.Interface, namespace, name, size, storageClass, accessMode string, blockVolume bool) (*v1.PersistentVolumeClaim, error) {
+	if !noCreate && len(size) == 0 {
```

Comment:
It's better to set required parameters here
https://github.com/kubevirt/kubevirt/pull/2574/files#diff-5193b6b6b04d7c6e897bb0c241ad8bfeR97
however I don't know if Cobra supports conditional required parameters, could you check that?
If not then I think we could validate that in `PreRun` [hook](https://github.com/spf13/cobra#prerun-and-postrun-hooks)

Reply:
yes, I can do a quick check and update here